src/object-identifier.scss

```diff
@@ -7,46 +7,31 @@ $block: #{$fd-namespace}-object-identifier;
   $fd-object-identifier-title-font-size: var(--sapFontLargeSize);
   $fd-object-identifier-title-font-weight: bold;
 
+  .#{$fd-namespace}-link {
+    font-size: inherit;
+    font-weight: inherit;
+  }
+
```

Comment:
We should not bring link component inside Object Identifier. You need to create a modifier class to overwrite the link properties.

Reply:
Done